File: FaceSample/app/src/main/java/com/regula/facesample/items/customization/fragment/HideCloseButtonFragment.java
package com.regula.facesample.items.customization.fragment;

import android.view.View;

import com.regula.facesdk.fragment.FaceDefaultUiFragment;

/**
 * Created by Sergey Yakimchik on 6.09.21.
 * Copyright (c) 2021 Regula. All rights reserved.
 */

public class HideCloseButtonFragment extends FaceDefaultUiFragment {

    @Override
    public View getCloseButton(View view) {
        View button = super.getCloseButton(view);
        button.setVisibility(View.INVISIBLE);
        return button;
    }
}

File: FaceSampleKotlin/app/src/main/java/com/regula/facesamplekotlin/fragments/HideCloseButtonFragment.kt
package com.regula.facesamplekotlin.fragments

import android.view.View
import com.regula.facesdk.fragment.FaceDefaultUiFragment

class HideCloseButtonFragment : FaceDefaultUiFragment() {

    override fun getCloseButton(view: View?): View {
        val button: View = super.getCloseButton(view)
        button.visibility = View.INVISIBLE
        return button
    }
}